modules/ossm-config-operator-infrastructure-node.adoc

```diff
@@ -18,29 +18,36 @@ If the operator will run on a worker node, skip this task.
 
 .Procedure
 
+. List the operators installed in the namespace by running the following syntax at the CLI:
```

Comment:
Do we use "syntax"? Don't we usually just say "command"?

Reply:
old habits die hard, as they say. Thanks for noting this!

Reply:
done